SplunkRumCrashReporting/SplunkRumCrashReporting/CrashReporting.swift

```diff
@@ -84,7 +94,9 @@ func loadPendingCrashReport(_ data: Data!) throws {
         exceptionType = report.exceptionInfo.exceptionName
     }
 
-    let oldSessionId = String(decoding: report.customData, as: UTF8.self)
+    let strArr = String(decoding: report.customData, as: UTF8.self).components(separatedBy: "|")
```

Comment:
This will break if the user specifiers a screen name containing a `|`.  Please use something sensible like URL/URI encoding (which in Swift is apparently named `addingPercentEncoding`) and the same keys we would report with the span.